criteria/document-objectives.md

```diff
@@ -29,7 +29,7 @@ There is an entry for the codebase objectives, mission or goal in the codebase d
 
 ## Policy makers: what you need to do
 
-* Add the policy objectives to the codebase documentation, for example in the `README`.
+* Add the policy objectives to the codebase documentation, for example in the `README`. Include relevant polices which impact the community, codebase, and development like value and ethics based polices, e.g.: equal opportunity.
```

Comment:
```suggestion
* Add the policy objectives to the codebase documentation, for example in the `README`. Include relevant polices which impact the community, codebase, and development like value and ethics based polices, like equal opportunity.
```

Reply:
Is using 'like' twice better than the e.g.? Perhaps we should spell out for example instead?